starter/story.js

```diff
@@ -0,0 +1,105 @@
+// PLEASE FORGIVE MY ENGLISH. THAT'S NOT THE POINT OF THIS ASSIGNMENT, RIGHT?
```

Comment:
Do you think storing these into an object will make it more logical if your game gets bigger?
e.g 
var scene = {
      scene1 : {
            response1: "Hey",
            response2: "Bye"
            },
      scene2 : {
             response1: "Thanx",
             response2: "Welcome
             }
}